kornia/geometry/boxes.py

```diff
@@ -340,10 +429,18 @@ def transform_boxes_(self, M: torch.Tensor) -> "Boxes":
         """Inplace version of :func:`Boxes.transform_boxes`"""
         return self.transform_boxes(M, inplace=True)
 
+    def _set_source_mode(self, mode: str) -> None:
+        """Set the default mode for functions like `to_tensor`."""
+        self._mode = mode
+
     @property
     def _is_batch(self) -> bool:
         return self._boxes.ndim == 4
 
+    @property
+    def _source_mode(self) -> str:
```

Comment:
```suggestion
    def mode(self) -> str:
```
no underscores for public members.